os/arch/arm/src/s5j/s5j_spi.c

```diff
@@ -574,5 +574,21 @@ struct spi_dev_s *up_spiinitialize(int port)
 	/* Disable Interrupts */
 	putreg32(0, &pSPIRegs->SPI_INT_EN);
 
-	return (struct spi_dev_s *)priv;
+	priv->spidev.priv = (void *)priv;
+
+	return &priv->spidev;
+}
+
+void s5j_spi_register(int bus)
+{
+	FAR struct spi_dev_s *spi;
+	char path[16];
+
+	spi = up_spiinitialize(bus);
+	if (spi != NULL) {
+		snprintf(path, 16, "/dev/spi-%d", bus);
```

Comment:
May be sizeof(path) would be better than hard coding 2nd parameter

Reply:
same as above